.githooks/pre-commit

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env sh
+RED="\033[1;31m"
+BLUE="\033[34m"
+GREEN="\033[1;32m"
+NOCOLOR="\033[0m"
```

Comment:
The colours seem to be printed out literally on powershell (e.g. `\033[34m(pre-commit) Running linters...\033[0m`). Should we remove these for now? (imo not a priority, but nice to have!)

This is the library we use for colouring our test scripts https://github.com/chalk/chalk, you can take a look at its implementation details if you would like to work this out as well!